var/spack/repos/builtin/packages/gettext/package.py

```diff
@@ -62,6 +62,7 @@ class Gettext(AutotoolsPackage, GNUMirrorPackage):
     # depends_on('cvs')
 
     conflicts("+shared~pic")
+    conflicts("%gcc4.8.5 operating_system=centos7", when="@0.22:")  # c++stdlib bug in centos7
```

Comment:
Why `os=centos7`? That looks over-constrained. What is the actual issue?

Reply:
I think the issue is that bundled gnulib is configured in not-c++11 mode, but then used with `g++ -std=gnu++11`.